walk-in-interview/src/main/java/com/google/job/data/JobQuery.java

```diff
@@ -0,0 +1,100 @@
+package com.google.job.data;
+
+/** Class for the job listings query using builder pattern. */
+public final class JobQuery {
+    private int minLimit = 0;
```

Comment:
How about unset value? E.g if I build a query and did not set the min or max and you don't apply those condition? Is it supported? Or are we requiring all query conditions to be set or else fallback to these default initialized value?

Reply:
If some query conditions aren't set, then they will be defaulted to these values